File: CS/Models/Product.cs
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;

public class Product {
    public int ProductID { get; set; }
    [Required(ErrorMessage = "Product Name is required")]
    public string ProductName { get; set; }
    [Required(ErrorMessage = "Unit Price is required")]
    public decimal? UnitPrice { get; set; }
    [Required(ErrorMessage = "Units On Order are required")]
    public short? UnitsOnOrder { get; set; }
}

File: VB/Models/Product.vb
Imports System.Web.Mvc
Imports System.ComponentModel.DataAnnotations

Public Class Product
	Public Property ProductID() As Integer
	<Required(ErrorMessage := "Product Name is required")>
	Public Property ProductName() As String
	<Required(ErrorMessage := "Unit Price is required")>
	Public Property UnitPrice() As Decimal?
	<Required(ErrorMessage := "Units On Order are required")>
	Public Property UnitsOnOrder() As Short?
End Class